src/workerd/api/node/tests/util-nodejs-test.js

```diff
@@ -25,24 +25,73 @@
 
 import assert from 'node:assert';
 import { mock } from 'node:test';
-import util, { inspect } from 'node:util';
+import util, {
+  inspect,
+  callbackify,
+  inherits,
+  promisify,
+  getCallSites,
+  emitExperimentalWarning,
+  stripVTControlCharacters,
+  styleText,
+} from 'node:util';
 
 const remainingMustCallErrors = new Set();
 function commonMustCall(f) {
   const error = new Error('Expected function to be called');
   remainingMustCallErrors.add(error);
-  return (...args) => {
+  return function (...args) {
     remainingMustCallErrors.delete(error);
-    return f(...args);
+    return f.call(this, ...args);
   };
 }
+function commonMustSucceed(fn, exact) {
+  return commonMustCall(function (err, ...args) {
+    assert.ifError(err);
+    if (typeof fn === 'function') return fn.apply(this, args);
+  }, exact);
+}
 function assertCalledMustCalls() {
   try {
     for (const error of remainingMustCallErrors) throw error;
   } finally {
     remainingMustCallErrors.clear();
   }
 }
+function commonMustNotCall(msg) {
```

Comment:
We don't implement common methods in workerd. We use 

```
const { promise, resolve } = Promise.withResolvers()
```

or 

```
const { mock } = require('node:test')
```

to get the call count of a function to validate if it runs multiple times.